src/web/mjs/connectors/MangaRolls.mjs

```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class KawaScans extends WordPressMadara {

```

Comment:
Class name does not match file name, suggested to change so they are in sync

Reply:
Fixed.